queries.js

```diff
@@ -92,4 +92,57 @@ db.restaurants.updateMany({cuisine: "American "}, { $set: { cuisine: "American"}
 db.restaurants.updateOne( { name: "Morris Park Bake Shop" }, { $set: { "address.street": "Calle falsa 123" } } )
 
 //11. Delete all the restaurants with address zipcode 10466.
-db.restaurants.deleteMany( { "address.zipcode": "10466" } )
\ No newline at end of file
+db.restaurants.deleteMany( { "address.zipcode": "10466" } )
+
+
+//Companies
+
+//1. Find all the companies that include 'Facebook' on the name field.
+
+db.companies.find( { name: "Facebook" } ).pretty()
+
+//2. Let's do it one more together:  ???
+
+//3. Find all the companies which category_code is 'web'. Retrive only their name field.
+
+db.companies.find( { category_code: "web" }, { name: 1, _id: 0 } ).pretty()
+
+//4. Find all the companies named "Twitter", and retrieve only their name, category_code and founded_year fields.
+
+db.companies.find( { name: "Twitter" }, { name: 1, category_code: 1, founded_year: 1, _id: 0 } ).pretty()
+
+//5. Find all the companies who have web as their category_code, but limit the search to 50 companies.
+
+b.companies.find( { category_code: "web" }, { name: 1, _id: 0 } ).limit(50).pretty()
```

Comment:
Un pequeño typo en el comienzo, simplemente falta la d de 'db'.

//5. Find all the companies who have web as their category_code, but limit the search to 50 companies.

db.companies.find( { category_code: "web" }, { name: 1, _id: 0 } ).limit(50).pretty()